swiperefresh/src/main/java/com/google/accompanist/swiperefresh/SwipeRefreshIndicator.kt

```diff
@@ -144,61 +102,108 @@ inline fun SwipeRefreshIndicator(
  */
 @Composable
 fun SwipeRefreshIndicator(
-    isRefreshing: Boolean,
-    offset: Float,
-    refreshOffset: Float,
+    state: SwipeRefreshState,
+    refreshTriggerDistance: Dp,
     modifier: Modifier = Modifier,
     scale: Boolean = false,
     arrowEnabled: Boolean = true,
     backgroundColor: Color = MaterialTheme.colors.surface,
     contentColor: Color = contentColorFor(backgroundColor),
     shape: Shape = MaterialTheme.shapes.small.copy(CornerSize(percent = 50)),
+    refreshingOffset: Dp = 16.dp,
     largeIndication: Boolean = false,
     elevation: Dp = 4.dp,
 ) {
-    val adjustedElevation = when (offset) {
-        0f -> 0.dp
-        else -> elevation
+    val adjustedElevation = when {
+        state.isRefreshing -> elevation
```

Comment:
Are these two conditions mutually exclusive?

Reply:
Nope, but they can be exclusive